kafka-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,123 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'Kafka_Alerts',
+        rules: [
+          {
+            alert: 'KafkaOfflinePartitonCount',
+            expr: |||
+              sum without(kafka_cluster) (kafka_controller_KafkaController_OfflinePartitionsCount{%(FilterSelector)s}) > 0
```

Comment:
sum without (kafka_cluster) would drop `kafka_cluster` making it impossible to use in description below.
What do we want to here?

Reply:
My bad, I misinterpreted the documentation. It should be correct now.